pb/event.go

```diff
@@ -89,11 +93,63 @@ func (e *PushEvent) Validate() error {
 	return nil
 }
 
-type RepositoryInfo = vcsurl.RepoInfo //TODO(mcuadros): improve repository references
+// RepositoryInfo containes information about a repository
+type RepositoryInfo struct {
+	CloneURL string
+	Host     string
+	FullName string
+	Owner    string
```

Comment:
Is there a reason to change vcsurl `Username` to `Owner`?

Reply:
mostly because for example `src-d` isn't a user but an organization. We will need to update lookout a little after this change anyway. So I decided to change it to more appropriate name.
It also removes confusion when you work with github. They also call it owner and username is the name of a requester.